packages/core/test/financial-templates/Liquidatable.js

```diff
@@ -1542,7 +1542,7 @@ contract("Liquidatable", function(accounts) {
           // State should be uninitialized as the struct has been deleted as a result of the withdrawal.
           // Once a dispute fails and the liquidator withdraws the struct is removed from state.
           ev.liquidationStatus.toString() == LiquidationStatesEnum.UNINITIALIZED &&
-          ev.settlementPrice.toString() == "0"
```

Comment:
oh were these cases where the value was empty before

Reply:
Not empty but 0 because the struct was deleted. These tests were therefore false positives because we fixed the the RHS of the assert.equal to match the (unexpected) 0 price

Reply:
These should be using JavaScript triple equals

Reply:
Yeah they all should good catch